docker-compose.production.yml

```diff
@@ -39,7 +39,9 @@ services:
 
   covers_nginx:
     profiles: ["ol-covers0"]
-    image: nginx:1.19.4
+    image: "${OLIMAGE:-openlibrary/olbase:latest}"
+    user: root
+    command: nginx -g "daemon off;"
```

Comment:
Note: This will make production covers use the new nginx with modified IP normalization... We'll likely need this to use the same start script as the other service. That way we'll also get log rotation.

Reply:
Fixed